pages/common/mc.md

```diff
@@ -0,0 +1,11 @@
+# mc
+
+> Midnight Commander, Command-Line Based File Manager.
+
+- Start mc in Terminal:
+
+`mc`
+
+- Navigate the folder structure using the arrow keys,the mouse or by typing the commands into the terminal:
```

Comment:
Thanks for fixing those other issues :-) It's probably just me being picky, but the same goes for here too - a single space after the comma would improve readability.